flow360/component/simulation/services.py

```diff
@@ -177,10 +185,18 @@ def get_default_params(
                         name="Freestream", surfaces=[Surface(name="placeholder2")]
                     ),  # to make it consistent with geo
                 ],
+                outputs=[
+                    SurfaceOutput(
+                        name="Surface output 1",
+                        entities=[Surface(name="placeholder")],
+                        output_fields=["Cp", "yPlus", "Cf", "CfVec"],
+                    ),
+                ],
```

Comment:
I think here we want to select "all" as well by default, right? The reason why we don't want to have all for wall, is that this setting will be always incorrect. But of output, I think this is ok. The down side is that we will export some data on freestream surfaces.

If we want all, then we can move this code outside the ifs to avoid duplication